### PR TITLE
[rush] Add RUSH_PARALLELISM environment variable

### PR DESCRIPTION
**Pull request by octogonz**
Fixes https://github.com/microsoft/rushstack/issues/1875

### CODE REVIEW

File: apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts [hunk 1]
```diff
@@ -156,8 +156,9 @@ export class BulkScriptAction extends BaseScriptAction {
         parameterLongName: '--parallelism',
         parameterShortName: '-p',
         argumentName: 'COUNT',
-        description: 'Specify the number of concurrent build processes'
-          + ' The value "max" can be specified to indicate the number of CPU cores.'
+        environmentVariable: 'RUSH_PARALLELISM',
```
Comment (iclanton) [inline apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts:159]: Should this just reference the enum from `EnvironmentConfiguration`?

Comment (octogonz) [reply apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts]: Fixed

File: apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap [hunk 2]
```diff
@@ -117,11 +117,14 @@ therein; a full rebuild is forced whenever the command has changed (e.g.
...
-                        of CPU cores. If this parameter omitted, the default 
-                        value depends on the operating system and number of 
-                        CPU cores.
+                        Specifies the maximum number of concurrent processes 
+                        to launch during a build. The COUNT should be a 
+                        positive integer or else the word \\"max\\" (which 
+                        specifies a count equal to the number of CPU cores). 
+                        If this parameter omitted, the default value depends 
+                        on the operating system and number of CPU cores. This 
+                        parameter may alternatively specified via the
```
Comment (sachinjoseph) [inline apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap:126]: This parameter may alternatively "be" specified

Comment (octogonz) [reply apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap]: Thanks, fixed

**Comment by octogonz** [comment]
This was published with Rush 5.23.4

**Pull request merged**